serialization/journal.proto

```diff
@@ -593,6 +593,22 @@ message GetBufferedLogging {
   optional Return return = 3;
 }
 
+message GetCelestialDegreesOfFreedom {
```

Comment:
Could we call this CelestialDegreesOfFreedom?  That would be consistent with the other celestial-related methods.